packages/strapi-types/index.ts

```diff
@@ -18,17 +23,45 @@ type BaseModel = {
   };
 };
 
-type ComponentModel = BaseModel & {};
+export type ComponentModel = Model & {};
+
+type Db = {
+  query: any;
+  getModel: any;
+};
+
+type Entity = {
+  [key: string]: Model;
+};
+
+export type Plugin = {
+  [key: string]: any;
+};
+
+export type ContentType = any;
 
-export type Strapi = {
+export class Strapi {
   components: {
     [key: string]: ComponentModel;
-  };
-  plugins: {
-    [key: string]: any;
-  };
+  } = {};
+  plugins: Plugin = {};
   contentTypes: {
-    [key: string]: any;
+    [key: string]: ContentType;
+  } = {};
+  db: Db = {
+    query: {},
+    getModel: {}
   };
-  query: (s: string) => any;
-};
+  query = (entity: string): Entity => {
+    return this.db.query(entity);
```

Comment:
??? I said that query should return real data, otherwise it will fail when using in tests